src/main/java/by/onliner/page/ShoppingCartPage.java

```diff
@@ -61,4 +77,62 @@ public String getEmptyCartMassage() {
         logger.info(String.format("Displayed [%s] message", emptyCartMessage.getText()));
         return emptyCartMessage.getText();
     }
+
+    /***
+     * Get the item name from shopping cart list.
+     * @return name of the item
+     */
+    @Step("Read the item name")
```

Comment:
I think it's better to use the word 'get' as 'read' is not really clear in the context of automated tests

Reply:
Fixed